src/common/evaluation/QA/eval_depthmap_models/src/evaluate.py

```diff
@@ -132,8 +134,26 @@ def get_prediction(MODEL_PATH, dataset_evaluation):
 
     print("Using {} artifact files for evaluation.".format(len(paths_evaluation)))
 
+    standing = load_model(FILTER_CONFIG.NAME)
+    new_paths_evaluation = paths_evaluation
+
+    if FILTER_CONFIG.SWITCH == 'ON':
+        new_paths_evaluation = []
+        exc = []
+        for p in paths_evaluation:
+            depthmap, targets, image = pickle.load(open(p, "rb"))
+            try:
+                image = utils.process_image(image)
+                if standing.predict(image) > .9:
+                    new_paths_evaluation.append(p)
+            except ValueError:
+                exc.append(image)
+
+    print(len(new_paths_evaluation))
+    print(len(paths_evaluation))
```

Comment:
To improve the readability of the code (`evaluate.py` is getting more and more complicated) we can move this logic into a function
```suggestion
    if FILTER_CONFIG.SWITCH == 'ON':
         new_paths_evaluation =  filter_dataset(paths_evaluation)
```